### ISSUE TITLE
ab-container block fails validation after changing alignment 

### ISSUE THREAD

**Issue by antpb** [ArrayHQ/atomic-blocks#39]
Hi! 

I noticed when playing around with the container block that changing alignment to not be centered causes a validation issue. If I find why, I can make a PR. :)

Using Version 1.4.2 . Here's the error:
```
Expected:

<div style="background-color:#fff;padding-left:5%;padding-right:5%;padding-bottom:20%;padding-top:20%;margin-top:0%;margin-bottom:0%" class="wp-block-atomic-blocks-ab-container alignundefined aligncenter ab-block-container"><div class="ab-container-inside"><div class="ab-container-image-wrap"><img class="ab-container-image has-background-dim-100 has-background-dim" src="http://antpb.staging.wpengine.com/wp-content/uploads/2018/06/Screenshot-2018-04-05-22.29.08.png" alt=""/></div><div class="ab-container-content" style="max-width:1600px"></div></div></div>

Actual:

<div style="background-color:#fff;padding-left:5%;padding-right:5%;padding-bottom:20%;padding-top:20%;margin-top:0%;margin-bottom:0%" class="wp-block-atomic-blocks-ab-container alignundefined ab-block-container"><div class="ab-container-inside"><div class="ab-container-image-wrap"><img class="ab-container-image has-background-dim-100 has-background-dim" src="http://antpb.staging.wpengine.com/wp-content/uploads/2018/06/Screenshot-2018-04-05-22.29.08.png" alt=""/></div><div class="ab-container-content" style="max-width:1600px"></div></div></div>
(anonymous) @ index.js?ver=1543977903:2
```

**Comment by sarahannnicholson** [comment]
This happens in the Accordion block as well

**Comment by mikemcalister** [comment]
The alignment and background color issue has been fixed in 1.5.0! Thanks for the heads up!
